traffic_ops/app/lib/API/User.pm

```diff
@@ -174,7 +179,9 @@ sub update {
 		registration_sent 		=> ( $params->{registrationSent} ) ? 1 : 0,
 		role 					=> $params->{role},
 		state_or_province 		=> $params->{stateOrProvince},
-		username 				=> $params->{username}
+		username 				=> $params->{username},
+		tenant_id 				=> $tenant_id
```

Comment:
Currently you may be able to add a root tenant. In a pending code, where I check authorization, you will not be able to.
Note that I see a root tenant as a "base parent tenant" - it has all the capabilities on its descendant tenants, but not on tenants not descendants of its.